### PR TITLE
STREAMS-2077 Explain how to accept the general conditions

### PR DESCRIPTION
**Pull request by cremond**
https://deploy-preview-93--streams-open-docs.netlify.app/docs/install/

Thank you for your contribution to the Axway Streams documentation.

## Describe the changes

Enter a brief description of your changes here to communicate to the maintainers what you changed and why.

## Checklist for contributors

Before submitting this PR, please make sure:

* [x] You have read the [contribution guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/)
* [x] You have signed the [Axway CLA](https://cla.axway.com/)
* [x] You have verified the technical accuracy of your change
* [x] You have verified that your change does not expose any sensitive information (passwords, keys, etc.)
* [x] You have followed the [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)  (unless this is is a Netlify CMS contribution)

_Put an x in the boxes that apply. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your change._

## Checklist for approvers

_This section is to be filled out by project maintainers only._

Before approving this PR, please make sure it:

* [ ] Does not have conflicts with the base branch
* [ ] Is believed to be accurate (technical accuracy to be checked with an SME for PRs originating from outside R&D)
* [ ] Follows [Axway style](https://techweb.axway.com/confluence/display/RIE/Style+guide)
* [ ] Follows [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)
* [ ] Follows [best practices](https://axway-open-docs.netlify.com/docs/contribution_guidelines/bestpracticedevdoc/)
* [ ] Does not contain typos, grammatical errors, etc.
* [ ] Does not expose any sensitive information
* [ ] Passes the Markdown linter rules (automated via CI check)
* [ ] Does not contain broken links

### CODE REVIEW

File: content/en/docs/install/_index.md [hunk 1]
```diff
@@ -52,6 +52,12 @@ There are different ways to manage your custom [Helm parameters](/docs/install/h
 
 Once your choice is made, we recommend you stick to it so that the [helm chart upgrade](/docs/install/upgrade/) is easier.
 
+### General Conditions for License/Subscription
+
+The Axway Products and/or Services shall be governed exclusively by the applicable General Terms and Conditions, located at [https://www.axway.com/en/legal/contract-documents](https://www.axway.com/en/legal/contract-documents).
+
+To accept them, you can set the helm value `acceptGeneralConditions` to `yes` and proceed with the installation.
```
Comment (guillaumelamirand) [inline content/en/docs/install/_index.md:59]: "you have to"

**Review by guillaumelamirand — CHANGES_REQUESTED** [review]

File: content/en/docs/install/_index.md (hunk 1 above)
Comment (cremond) [reply content/en/docs/install/_index.md]: done

File: content/en/docs/install/_index.md [hunk 2]
```diff
@@ -52,6 +52,12 @@ There are different ways to manage your custom [Helm parameters](/docs/install/h
 
 Once your choice is made, we recommend you stick to it so that the [helm chart upgrade](/docs/install/upgrade/) is easier.
 
+### General Conditions for License/Subscription
```
Comment (guillaumelamirand) [inline content/en/docs/install/_index.md:55]: Réduire le niveau du titre a ##

Comment (cremond) [reply content/en/docs/install/_index.md]: done

**Review by guillaumelamirand — APPROVED** [review]

**Comment by guillaumelamirand** [comment]
@andreamussap Could you please review and merge this one ? Thanks

**Pull request merged**